yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can you remove this file from the commit? The `yarn.lock` file already exists under `frontend`. It looks like you accidentally ran a `yarn` command from the repo root directory.